ansible/configs/ocp-workshop/destroy_env.yml

```diff
@@ -33,6 +33,29 @@
         - s3_result is not succeeded
         - cloud_provider == 'ec2'
 
+    - name: Delete ec2 cloud provider created resources
+      environment:
+        AWS_ACCESS_KEY_ID: "{{ aws_access_key_id }}"
+        AWS_SECRET_ACCESS_KEY: "{{ aws_secret_access_key }}"
+        AWS_DEFAULT_REGION: "{{ aws_region_final|d(aws_region) }}"
+      block:
+      - name: Finding ELBs
+        action:
+          module: ec2_elb_facts
+        register: elbs
+
+      - set_fact:
+          elb_names: "{{ elbs | json_query(query) }}"
+        vars:
+          query: "elbs[?tags.\"kubernetes.io/cluster/{{ guid }}\" == 'owned'].name"
+
+      - name: Deleting ELBs provisioned by cloud provider
+        ec2_elb_lb:
+          name: "{{ item }}"
+          state: absent
+        loop: "{{ elb_names }}"
+      when: cloud_provider == 'ec2' and install_cloud_provider|bool
+
```

Comment:
what about other kind of resources ? Are you sure this will cleanup **everything** ?

persistent volumes as EBS ?

Reply:
@fridim That's a good point. I have not thought about EBS volumes...I'll investigate more...